flow-typed/types/playback-config.js

```diff
@@ -12,5 +12,6 @@ declare type PKPlaybackConfigObject = {
   muted: boolean,
   pictureInPicture: boolean,
   streamPriority: Array<PKStreamPriorityObject>,
-  preferNative: PKPreferNativeConfigObject
+  preferNative: PKPreferNativeConfigObject,
+  inBrowserFullscreenForIOS: boolean
```

Comment:
add to `player-config.js` with default value

Reply:
Added